onnxruntime/core/session/inference_session.cc

```diff
@@ -289,7 +301,9 @@ common::Status InferenceSession::Load(const void* model_data, int model_data_len
       return Status(common::ONNXRUNTIME, common::INVALID_PROTOBUF,
                     "Failed to load model because protobuf parsing failed.");
     }
-
+#ifdef ENABLE_PYOP
```

Comment:
Let's call this macro something like LANGUAGE_INTEROP_SUPPORT and then call LoadInteropOps() which in turn calls LoadPyOp(). Both LoadInteropOps() and LoadPyOp() can reside in a folder called language_interop under 'session' dir and implemented in separate files so that say a JavaOp can just be implemented independently in a separate file called by the upper level wrapper (LoadInteropOps)

Reply:
Done.